src/app/settings/settings.component.ts

```diff
@@ -177,5 +180,8 @@ export class SettingsComponent extends RouterEnter {
       });
     }
   }
-
+  async goToSetting() {    
+    await PusherBeams.goToAppSetting();
```

Comment:
@ronakamani in the settingComponent, you are still using the same function served from PusherBeams plugin.